bootstrap.php

```diff
@@ -295,36 +295,12 @@ function (Application $app) {
                     'prefix' => $cacheType . '_',
                 ]
             );
-            $cache = new Doctrine\Common\Cache\PredisCache($redisClient);
 
-            return $cache;
+            return new Doctrine\Common\Cache\PredisCache($redisClient);
         };
     }
 );
 
-$app['cache-filesystem'] = $app->share(
-    function () {
-        $baseCacheDirectory = __DIR__ . '/cache';
-
-        return function ($cacheType) use ($baseCacheDirectory) {
-            $cacheDirectory = $baseCacheDirectory . '/' . $cacheType;
-
-            $cache = new \Doctrine\Common\Cache\FilesystemCache($cacheDirectory);
-
-            return $cache;
-        };
-    }
-);
-
-$app['cache'] = $app->share(
-    function (Application $app) {
-        $activeCacheType = $app['config']['cache']['active'] ?: 'filesystem';
```

Comment:
I'm wondering what would be a good way to have a reminder to also remove this from the `appconfig` once this change has been deployed. Any suggestions how we can keep track of that?

Reply:
It's technically not _necessary_ to remove it, but if we don't we end up with a lot of obsolete config which will get confusing in the long run.

Reply:
And if we remove it from the `appconfig` right now, it will cause a deploy of the config and suddenly revert it to `filesystem` until this change is also deployed.

Reply:
We can keep a simple todo list with items to process "after next deploy"?